example/workflows/sips.yml

```diff
@@ -30,6 +30,7 @@ SyncGranule:
         fileStagingDir: 'custom-staging-dir'
         downloadBucket: '{$.cumulus_meta.system_bucket}'
         duplicateHandling: '{$.meta.collection.duplicateHandling}'
+        reingestGranule: '{$.meta.reingestGranule}'
```

Comment:
Hmmmm I don't like that this is something that needs to be configured by the user. We don't want them to have to go update all their workflows.

I don't know the _best_ way to do this, but I think we need a mechanism for tasks to be able to grab things out of cumulus_meta. I know we do some of that in the message adapter but don't know if that's the right answer.

Off the top of my head, I'm thinking we could grab it out of the event before calling the message adapter. Or we could update the message passed to the message adapter to also pull out `reingestGranule`.

I'm also thinking `reingestGranule` should go in `cumulus_meta`, but it's probably fine in `meta`.

Reply:
Should we solve that in a different ticket?